hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestListKeys.java

```diff
@@ -67,6 +67,9 @@ public class TestListKeys {
   private static String omId;
 
   private static OzoneBucket legacyOzoneBucket;
+
+  private static OzoneBucket fsoOzoneBucket;
```

Comment:
There is already a TestListKeysWithFSO.java class for FSO test cases, It also has some test-cases with shallow=false. Can we shift the test-cases there that are not covered?

Reply:
Good idea.